### PR TITLE
feat(directory): Add services to biobank detail view

### PR DESCRIPTION
**Pull request by connoratrug**
Closes https://github.com/molgenis/molgenis-emx2/issues/4547

### What are the main changes you did
- Add service details to biobak details request
- Show services connected to biobank on biobank details page
- Add tab component to toggle between collections and services on biobank details page
- Collapse ( hide ) service details by default on biobank details view 

### How to test
- see issue and unit test

### Checklist
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Comment by esthervanenckevort** [comment]
> > I found that the biobank details page does not show if there are no services associated to the biobank. On the main page the services tab is then also empty. I didn't directly see an error in the browser logs.
> 
> What would you like to see and where , can you give an example using the demo data ?

Unfortunately the demo data didn't cover this case, so I created a new biobank on the PR server without any associated services. I think we should add one to the demo data (or at least a test) to capture this case, which will occur immediately when rolling out this release (because biobanks can only add their services afterwards).

Similar to the case when there are no collections I would expect a text that says: "This biobank doesn't offer any services yet." 


> 
> > Two other minor items:
> > 
> > * The style of the add/remove button on the biobanks details page is different for the services tab compared to the collections tab. In the collections tab the button is a little bit rounder
> 
> yes, the collections use a more custom component, which style do you prefer ? ( and why )

Stylistically I don't have a strong preference for one or the other, but the more rounded buttons are in the same style as the rest of the application, so for consistency I have a small preference for that.

> 
> > * The collections list in the biobanks details page hides the details of the collections behind a show more details, while the services show all the details. For long lists this behaviour for the collections list is desirable and I think it should be consistent between the two tabs.
> 
> For a small list, the collapsed style feels a bit silly, most of the list in prod are small ( < 10). I'll wrap it in a collapse component and maybe we can use this dynamically depending on the list size ( is size < 3 wrap using collapse ). Consistency can help structure data but is not an endgoal in and of itself

I fully agree with that. Making this dynamic would make sense, but I think it would be if size > 3 wrap using collapse, right?

Another thing I noticed is that the collections link to the collections details page but the services don't. Another small consistency thing.

File: apps/directory/tests/show-services-on-biobank-detail.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
```
Comment (chinook25) [inline apps/directory/tests/show-services-on-biobank-detail.spec.ts:1]: Run formatter on this file

**Review by chinook25 — APPROVED** [review]
Code looks good to me and didn't find issues running it. Just one minor comment.

File: apps/directory/tests/show-no-service-message.spec.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { test, expect } from "@playwright/test";
+import { getAppRoute } from "./getAppRoute";
+
+test("show message if biobanik has no services", async ({ page }) => {
```
Comment (esthervanenckevort) [inline apps/directory/tests/show-no-service-message.spec.ts:4]: Minor typo biobanik instead of biobank

**Review by esthervanenckevort — APPROVED** [review]

**Pull request merged**